src/catalogue/go.mod

```diff
@@ -3,6 +3,7 @@ module mushop/catalogue
 go 1.13
 
 require (
+	github.com/DATA-DOG/go-sqlmock v1.4.1
```

Comment:
Is this something that could also be used for “mock” mode? Thereby allowing the API to use all actual micro services, with mocked backing services?

Reply:
Yes, I'm planning something similar.